platform/admin/ui/src/main/webapp/js/models/Installer.js

```diff
@@ -139,7 +140,22 @@ define([
                     url: that.installUrl + 'admin-post-install-modules/',
                     dataType: 'JSON'
                 }).then(function(){
-                    location.reload();
+                    if (reboot) {
+                        $.ajax({
+                            type: 'GET',
+                            url: that.rebootUrl,
+                            dataType: 'JSON'
+                        }).done(function () {
+                            window.setTimeout(function () {
+                              window.location.href = that.get("redirectUrl");
+                            }, 30000);
+                        }).fail(function () {
+                            alert("Error trying to restart system. Please manually restart the system.");
```

Comment:
So far we have not used `alert` in our web apps.  Usually we will display the error on the page with the error styling.  This `alert` should be replaced and instead look like the configuration model and insecure defaults errors.